spec/features/prospects_management_spec.rb

```diff
@@ -3,14 +3,60 @@
 
 RSpec.describe "Prospects management" do
   scenario do
-    create(:prospect, name: "Iron Man", description: "Does want suit upgrade", target: true)
+    create(:prospect, name: "Iron Man", description: "Does want suit upgrade", target: true, uid: 87)
+
+    allow(ProspectApiWrapper).to receive(:get_prospects) do
+      [
+        { name: "Captain America",
+          description: "Needs new shield",
+          target: true,
+          uid: 88 }
+      ]
+    end
 
     behavior "Admin sees prospects" do
       admin_ux.navigate_to_prospects
 
       expect(admin_ux).to have_selector("h1", text: "Prospects")
       expect(admin_ux).to have_table_row_contents("Iron Man", "Does want suit upgrade", "true", position: 1)
     end
+
+    behavior 'Admin downloads prospects with "Import Prospects" button' do
+      admin_ux.download_prospects
+
+      expect(admin_ux).to have_flash_notice("1 new prospect was imported")
+
+      expect(admin_ux).to have_table_row_contents("Captain America", "Needs new shield", "true", position: 1)
+      expect(admin_ux).to have_table_row_contents("Iron Man", "Does want suit upgrade", "true", position: 2)
+    end
+
+    allow(ProspectApiWrapper).to receive(:get_prospects) do
```

Comment:
prefer using `{}`. There is a suggestion somewhere to use `{ }` for blocks with one expression and `do...end` for blocks with multiple expressions, regardless of line count